endpoint_poses:                                                      # This must match the name of the node that gets them
  ros__parameters:
    names: ["up", "down", "left", "right", "fore", "aft", "round"]   # We need to declare them here so we can search them by name
    up:
        poses: [0.0, 0.0, 0.2, 0.0, 0.0, 0.0]
        times: [2.0]
        reference: "relative"
    down:
        poses: [0.0, 0.0, -0.2, 0.0, 0.0, 0.0]
        times: [2.0]
        reference: "relative"
    left:
        poses: [0.0, 0.2, 0.0, 0.0, 0.0, 0.0]
        times: [2.0]
        reference: "relative"
    right:
        poses: [0.0, -0.2, 0.0, 0.0, 0.0, 0.0]
        times: [2.0]
        reference: "relative"
    fore:
        poses: [0.2, 0.0, 0.0, 0.0, 0.0, 0.0]
        times: [2.0]
        reference: "relative"
    aft:
        poses: [-0.2, 0.0, 0.0, 0.0, 0.0, 0.0]
        times: [2.0]
        reference: "relative"
    round:
        poses: [0.55, -0.25, 0.7, 0.0, 1.5708, 0.0,
                0.55,  0.15, 0.7, 0.0, 1.5708, 0.0,
                0.55,  0.15, 0.5, 0.0, 1.5708, 0.0,
                0.55, -0.25, 0.5, 0.0, 1.5708, 0.0,
                0.55, -0.25, 0.7, 0.0, 1.5708, 0.0]
        times: [2.0, 4.0, 6.0, 8.0, 9.0]
        reference: "absolute"
